chapters/package-information.md

```diff
@@ -38,7 +38,7 @@ EXAMPLE 2 RDF: Property `spdx:name` in class `spdx:Package`
 
 ### 7.2.1 Description
 
-Uniquely identify any element in an SPDX document which may be referenced by other elements. These may be referenced internally and externally with the addition of the SPDX document identifier. The metadata for the SPDX identifier field is shown in Table 14.
+Uniquely identify any element in an SPDX document which may be referenced by other elements. These may be referenced internally and externally with the addition of the SPDX document identifier. The metadata for the package SPDX identifier field is shown in Table 14.
 
 **Table 14 — Metadata for the SPDX identifier field**
```

Comment:
```suggestion
**Table 14 — Metadata for the package SPDX identifier field**
```

Maybe also change this table heading for consistency?

Reply:
Good idea. I’ll include that change in the next force push.